toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -348,6 +348,18 @@ please refer to the [overlay type](#overlay-type) section.
 - `hashPartition`: A partition used exclusively for storing a calculated hash
   tree.
 
+- `corruptionOption`: Optional. Specifies the behavior in case of detected
+  corruption. This is configurable with the following options:
+  - `CorruptionOptionDefault`: Without specifying corruption behaviors, or
```

Comment:
Specify the string values here. The names of enum values in the Go code are an implementation detail.

Reply:
Fixed, thanks.